SIT.Manager.Avalonia/Services/InstallerService.cs

```diff
@@ -278,6 +287,76 @@ public class InstallerService(IBarNotificationService barNotificationService,
             return string.Empty;
         }
 
+        public async Task<List<GithubRelease>> GetServerReleases() {
+            List<GithubRelease> githubReleases = [];
+            try {
+                string releasesJsonString = await _httpClient.GetStringAsync(@"https://api.github.com/repos/stayintarkov/SIT.Aki-Server-Mod/releases");
+                githubReleases = JsonSerializer.Deserialize<List<GithubRelease>>(releasesJsonString) ?? [];
+            }
+            catch (Exception ex) {
+                githubReleases = [];
+                _logger.LogError(ex, "Failed to get server releases");
+            }
+
+            List<GithubRelease> result = [];
+            if (githubReleases.Any()) {
+                foreach (GithubRelease release in githubReleases) {
+                    var zipAsset = release.assets.Find(asset => asset.name.EndsWith(".zip"));
+                    if (zipAsset != null) {
+                        Match match = ServerReleaseVersionRegex().Match(release.body);
+                        if (match.Success) {
+                            string releasePatch = match.Groups[1].Value;
+                            release.tag_name = release.name + " - Tarkov Version: " + releasePatch;
```

Comment:
Can we replace this with string interpolation plz

Reply:
Yep done